runtime_builders/publish_runtime.py

```diff
@@ -19,6 +19,7 @@
 import json
 import logging
 import os
+import regex
```

Comment:
the std library `re` doesn't support variable length lookbehind, while this one does. the regex that I'm using to parse out the tags from image names wouldn't work in the std library.